packages/oui-select/src/index.spec.js

```diff
@@ -49,6 +49,23 @@ describe("ouiSelect", () => {
             expect(getDropdown(element)).toBeDefined();
         });
 
+        it("should display the select component without passing match attribute", () => {
```

Comment:
The test should be "should display all the choices (stringArray)" and assert on choices.
This line isn't tested "matchElement.html("{{$select.selected}}");".

Reply:
NVM I saw the new test should display all the choices (objectArray)
This one isn't useful now